subsys/net/lib/nrf_cloud/src/nrf_cloud_agps.c

```diff
@@ -573,6 +626,17 @@ int nrf_cloud_agps_process(const char *buf, size_t buf_len)
 		}
 	}
 
+#if defined(CONFIG_NRF_CLOUD_AGPS_FILTERED)
+/**
+ * In filtered mode, because fewer than the full set of ephemerides is sent to
+ * the modem, determine here if we correctly received them from the cloud and
+ * sent them to the modem.
+ */
```

Comment:
```suggestion
	/**
	 * In filtered mode, because fewer than the full set of ephemerides is sent to
	 * the modem, determine here if we correctly received them from the cloud and
	 * sent them to the modem.
	 */
```